preprocess.py

```diff
@@ -60,13 +60,17 @@ def process_wav(path):
         with open(f'{paths.data}text_dict.pkl', 'wb') as f:
             pickle.dump(text_dict, f)
 
-    simple_table([('Sample Rate', hp.sample_rate),
-                  ('Bit Depth', hp.bits),
-                  ('Mu Law', hp.mu_law),
-                  ('Hop Length', hp.hop_length),
-                  ('CPU Count', cpu_count())])
+    n_processes = min(8, cpu_count())
```

Comment:
Whats the motivation for limiting to 8 cores? My workstations have 40 and 36, wouldn't it make sense to utilize them? If the goal is to avoid using all the theads, why not keep the OS and maybe one other thread free by doing `max(1, cpu_count()-2)`?

Reply:
I'm thinking about situations where there are multiple users of a workstation. I think 8 is plenty - wouldn't there be diminishing returns after that with synchronisation overhead and whatnot?

Reply:
Hard drive: 
When checking to see how many wav files (13100 files total) are processed in two minutes time, I get the following: 
1/36 cores: 1150
4/36 cores: 2201
8/36 cores: 2932
18/36 cores: 3983
24/36 cores: 4418
35/36 cores: 5067
![cores vs processing time](https://user-images.githubusercontent.com/6969415/61733474-c1a17780-ad34-11e9-946b-687b0d166b6c.png)

I think with that big of a speedup, it makes sense to let the number of processes scale with the number of cores. For multi-user workstations, the OS scheduler should give the threads priority based on the niceness versus other users. If you're really worried about being kind to your coworkers, I think a command line flag should do the trick. 

I do agree that saving one core for the OS to ensure responsiveness is a good call though. I propose we do `n_processes = max(1, args.num_cores if args.num_cores else cpu_count()-1)`

Reply:
Yeah that's good.